cmd/satellite/main.go

```diff
@@ -415,6 +422,12 @@ func cmdNodeUsage(cmd *cobra.Command, args []string) (err error) {
 	return generateNodeUsageCSV(ctx, start, end, file)
 }
 
+func cmdStripeCustomer(cmd *cobra.Command, args []string) (err error) {
+	ctx, _ := process.Ctx(cmd)
+
+	return generateStripeCustomers(ctx)
```

Comment:
I would make it a grpc call to the satellite

Reply:
This is only a one of command and can likely be removed with the next release.

Reply:
it's applicable to customer payments setup as well, it is more straightforward and easier to manage, as satellite has all appropriate services. Again, that is just a suggestion

Reply:
Going forward each new signup will have a stripe account already. This part only ensures, we have one for users pre billing.